src/client/components/componentWithSliders/allSliders/allBlocks.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import './alllBlocks.css';
+
+import BlockOfSliders from './blockOfSliders';
+import PlayButton from './playButton';
+import StopButton from './stopButton';
+
+import BLOCKS from '../../../helpers/blocks';
+import Pizzicato from "pizzicato";
+
+
+class AllBlocks extends Component {
+
+  constructor(props, context){
+    super(props, context);
+    this.state = {
+      sound: new Pizzicato.Sound({
+        source: 'file',
+        options: { path: require('../../../sounds/sound.mp3') }
+        },
+        () => BLOCKS.forEach(({createEffect}) => {
+          this.state.sound.addEffect(createEffect);
+            })
+      ),
+      playing: false,
+    };
+  }
+
+  play = () => {
```

Comment:
as i understood this function will play smth on first call, but on second call it will pause smth. It would be better to rename this function to understand what this function is doing from its name.

Reply:
fixed